### ISSUE TITLE
Prediction with RadiusNeighborsRegressor

### ISSUE THREAD

**Issue by howardwu8** [vincenzocoia/BAIT509#14]
Hello professor:

When I use Python sklearn package, I come across several issues:

In the dataset, some of the data for 'age' and 'fare' are missing. As we learned from other classes, there are several ways to deal with missing data. While you might mention to some of my classmates that just delete the full rows of the missing data, I believe there are other ways such as full with 0 or mean value to handle the missing data. Please specify your requirement or it's up to us to choose.

I went ahead with deleting the full rows of the missing data, but then in the RadiusNeighborsRegressor part, after training the model from training set, and predict it using test data, the predictions including numerous 'NaN', and as bandwidth increases, the number of 'NaN' decreases. Now I face such situation: As bandwidth increases., the test set mse decreases and becomes lower than the mse of training set.. which does not make sense to me..

Please kindly advise what we should handle both the missing data from dataset, and missing data from prediction.
